pkg/controller/registrysyncer/registrysyncer_test.go

```diff
@@ -305,6 +321,30 @@ func TestReconcile(t *testing.T) {
 					return fmt.Errorf("actual does not match expected, diff: %s", diff)
 				}
 
+				actualImageStreamFin := &imagev1.ImageStream{}
+				if err := apiCIClient.Get(ctx, types.NamespacedName{Name: "applyconfig", Namespace: "ci"}, actualImageStreamFin); err != nil {
+					return err
+				}
+				expectedImageStreamFin := &imagev1.ImageStream{
```

Comment:
Please avoid abbreviations like `Fin` and spell out what you mean (`WithFinalizer`?), you will have forgotten what they mean in two weeks

Reply:
Removed.